common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationRequest.java

```diff
@@ -151,10 +138,15 @@ public Uri getAuthorizationRequestAsHttpRequest() throws UnsupportedEncodingExce
             uriBuilder.appendQueryParameter(entry.getKey(), entry.getValue());
         }
 
-        for (Map.Entry<String, String> entry : ObjectMapper.deserializeQueryStringToMap(mSliceParameters).entrySet()) {
+        for (Map.Entry<String, String> entry : mFlightParameters.entrySet()) {
             uriBuilder.appendQueryParameter(entry.getKey(), entry.getValue());
         }
 
+        if(mSlice != null){
+            uriBuilder.appendQueryParameter("slice", mSlice.getSlice());
```

Comment:
nit: Can the same `"slice"` String constant be used here and in `AzureActiveDirectorySlice.java`?